src/ILCompiler.Compiler/src/Compiler/DependencyAnalysis/ReadyToRunHelperNode.cs

```diff
@@ -22,11 +22,11 @@ public enum ReadyToRunHelperId
         GetThreadStaticBase,
         DelegateCtor,
         ResolveVirtualFunction,
-
```

Comment:
This is separated because the rest of the helpers are only used for generic lookups. Not adding a comment is my fault. Can you add one please (`// Following helpers are used for generic lookups only`)?